x-pack/plugins/observability_solution/synthetics/server/saved_objects/synthetics_monitor.ts

```diff
@@ -200,6 +200,9 @@ export const getSyntheticsMonitorSavedObjectType = (
         id: {
           type: 'keyword',
         },
+        config_id: {
```

Comment:
It looks like https://github.com/elastic/kibana/pull/167705 has landed in serverless (up to prod-canary) as of yesterday. If we want to be extra cautious, we may want to just make sure it gets deployed to production (scheduled for today) before merging this PR. This would mean that `config_id` is already explicitly included in the new AAD include list and adding the mapping should be ok in the next release.
cc: @kc13greiner